### PR TITLE
Adding user level and task level gadgets

### PR DESCRIPTION
**Pull request by dewmini**
Adding user level and task level gadgets for PC Analytics

### CODE REVIEW

File: modules/extensions/apps/portal/store/carbon.super/fs/gadget/avg_execution_time_vs_task_id/js/actions.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+                $.getJSON("/portal/store/carbon.super/fs/gadget/avg_execution_time_vs_task_id/js/meta-data-avgExeTimeVsTaskID.json.js", function (result) {
+                    $.each(result, function (i, field) {
+                        jsonObj.push(field);
+                        drawGraph();
+                    });
+                });
+            }
+            else{
+                console.log('Empty User ID list.');
```
Comment (tharikaGitHub) [inline modules/extensions/apps/portal/store/carbon.super/fs/gadget/avg_execution_time_vs_task_id/js/actions.js]: Shall we remove the use of console.log() here? Not all browsers support console.log(). You can use alert() instead. Or use a dummy placeholder to render if the console is undefined.

File: modules/extensions/apps/portal/store/carbon.super/fs/gadget/avg_execution_time_vs_task_id/js/actions.js [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+                }
+            },
+            error: function (xhr, status, error) {
+                var errorJson = eval("(" + xhr.responseText + ")");
+                alert(errorJson.message);
+            }
+        });
+    } else {
+        console.log('Empty User ID list.');
+        alert("User ID list is empty.");
```
Comment (tharikaGitHub) [inline modules/extensions/apps/portal/store/carbon.super/fs/gadget/avg_execution_time_vs_task_id/js/actions.js]: Both console.log and alert are not needed. You can keep alert() here. They do nearly the same.

Comment (dewmini) [inline modules/extensions/apps/portal/store/carbon.super/fs/gadget/avg_execution_time_vs_task_id/js/actions.js]: Fixed in https://github.com/wso2/product-pc/commit/e931bb704e9065cf57f08f61619a45a4beecb5c7

File: modules/extensions/apps/portal/store/carbon.super/fs/gadget/avg_execution_time_vs_task_id/js/actions.js (hunk 1 above)
Comment (dewmini) [inline modules/extensions/apps/portal/store/carbon.super/fs/gadget/avg_execution_time_vs_task_id/js/actions.js]: Fixed in https://github.com/wso2/product-pc/commit/e931bb704e9065cf57f08f61619a45a4beecb5c7

**Pull request merged**